examples/Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[workspace]
+members = [
+     "chacha20",
```

Comment:
What about kyber-serialize?

Reply:
Yes, we should include it at some point, but for this PR I just want to introduce `core` + a few examples.
Then I'll make a follow-up PR with more examples (and better CI).